src/services/DotbitService.ts

```diff
@@ -104,30 +106,30 @@ export class DotbitService extends NamingService {
         symbol: addrs[0].subtype.toUpperCase()
       }
     }
-    this.throwError(`${name}'s address is not found`, AllDIDErrorCode.RecordIsNotFound)
+    return null
```

Comment:
Returning value here is different from ENS when the result is empty.

https://github.com/dotbitHQ/denames/pull/17/files#diff-e5cb1ecfdc62099b3325ef896d21dd7308919af8842372d69bfaf01d09c58760R201